shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -39,13 +39,23 @@
 public class SummaryCommand extends AbstractCommand {
 
   @Override
-  public int run(CommandLine cl) throws IOException {
+  public int run(CommandLine cl) throws UnavailableException, IOException {
```

Comment:
UnavailableException extends IOException, so we don't need to declare it here

Reply:
Thanks for that information. 
Removed UnavailableException, thanks